test/autoTestGAP.sh

```diff
@@ -0,0 +1,148 @@
+set -e
+set -u
+
+function preparePipeline(){
+
+	local _workflowType="${1}"
+
+	local _projectName="NIST_TRIO"
+	rm -f ${tmpfolder}/logs/${_projectName}/run01.pipeline.finished
+	echo "TMPFOLDER: ${tmpfolder}"
+	pwd
+	rsync -r --verbose --recursive --links --no-perms --times --group --no-owner --devices --specials ${pipelinefolder}/test/rawdata/203693990030 ${tmpfolder}/rawdata/array/GTC/
+
+	if [ -d "${tmpfolder}/generatedscripts/${_projectName}" ]
+	then
+		rm -rf "${tmpfolder}/generatedscripts/${_projectName}/"
+	fi
+
+	if [ -d "${tmpfolder}/projects/${_projectName}" ]
+	then
+		rm -rf "${tmpfolder}/projects/${_projectName}/"
+	fi
+
+	if [ -d "${tmpfolder}/tmp/${_projectName}" ]
+	then
+		rm -rf "${tmpfolder}/tmp/${_projectName}/"
+	fi
+	mkdir "${tmpfolder}/generatedscripts/${_projectName}/"
+
+	cp "${pipelinefolder}/templates/generate_template.sh" "${tmpfolder}/generatedscripts/${_projectName}/generate_template.sh"
+
+sudo -u umcg-envsync bash -l << EOF
+
+id
+export SOURCE_HPC_ENV="True"
+. ~/.bashrc
+module load depad-utils
+#module list
+hpc-environment-sync.bash -m GAP/betaAutotest
+exit
+
+EOF
+
+	module load GAP/betaAutotest
```

Comment:
als je in de module file van de betaAutotest er in zet dat ie ook compute mee moet laden hoef je die niet meer apart te module loaden